applications/ContactStructuralMechanicsApplication/README.md

```diff
@@ -1,55 +1,56 @@
-## Contact Structural Mechanics Application 
+# Contact Structural Mechanics Application 
  
-The Contact Structural Mechanics Application contains the contact mechanics implementations that can be used by the Structural Mechanics Application within Kratos Multiphysics. 
```

Comment:
Structural and ConstitutiveLawsApp I would add

Reply:
Done